build.xml

```diff
@@ -11,8 +11,8 @@
 
   <!-- defaults -->
   <property name="idea.product" value="ideaIC"/>
-  <property name="idea.version" value="2016.3.1"/>
-  <property name="dart.plugin.version" value="163.10154.19"/>
+  <property name="idea.version" value="2016.3.2"/>
```

Comment:
As per our conversation, I'll change this back and opt for a reflective solution.